README.md

```diff
@@ -146,11 +146,11 @@ To add a new model, you have to modify/create five files (for further details, a
         case ModelIDs::YourModel: return std::make_unique<Class_YourModel>(); break;
 
 ### Generate the C++ code for a model
-We provide currently two methods to generate the tensors and calculate the counter terms for a new model.
+We provide currently three methods to generate the tensors and calculate the counter terms for a new model.
 
 1. At `tools/ModelGeneration/Maple` we provide the maple Worksheet `CreateModel.mw` which you can use to implement your model and get the tensors.
-2. At `tools/ModelGeneration/sympy` we provide a setup using only `python3` with `sympy` (at least version 1.10!, if your packet manager only has an older installed, e.g. ubuntu 20.04 only has v1.6, then you have to install v1.10 or up with pip). Here we provide two examples, `SM.py` and `G2HDM.py` (generic 2HDM) which both use the `ModelGenerator.py` module to calculate the tensors and CT. You can get the CT using `python3 SM.py --show ct` and the tensors by calling `python3 SM.py --show tensors`. If your counterterms don't have a unique solution, then the solution space will be shown to you and you have to add additional equations until you have a unique solution (e.g. in the G2HDM example).
-3. To show the simplified tree-level and counterterm potentials, you can use `python3 SM.py --show treeSimpl` and `python3 SM.py --show CTSimpl`.
+2. At `tools/ModelGeneration/Mathematica` we provide a Mathematica framework to implement your model.
```

Comment:
Any reason why you made it as number 2? 
We could try to reorder them in a way that the numbering represents the recommend method.

Reply:
No reason, I reordered them as 1. `python`, 2. `Mathematica`, 3. `Maple` (from open to closed source, most to least widely used).